buildtools/fsyacc/fsyacc.fs

```diff
@@ -56,12 +56,15 @@ let main() =
                | Some outputLogName -> new FileLogger(outputLogName) :> Logger
                | None -> new NullLogger() :> Logger
   let compiledSpec = compileSpec spec logger
-  printfn "        building tables"
-  printfn "        %d states" compiledSpec.states.Length; 
-  printfn "        %d nonterminals" compiledSpec.gotoTable.[0].Length; 
-  printfn "        %d terminals" compiledSpec.actionTable.[0].Length; 
-  printfn "        %d productions" compiledSpec.prods.Length; 
-  printfn "        #rows in action table: %d" compiledSpec.actionTable.Length; 
+
+  // Printing is nopt useful for day-to-day compiler dev and was just taking time and space in output.
+  // Enable if needed.
+  //printfn "        building tables"
```

Comment:
I'm not particularly fond that you are not introducing a change in https://github.com/fsprojects/FsLexYacc first.
By doing this you are deviating from the source and nobody will ever dare to sync them again in the future.

Also, make this configurable, and disable it by default.

Reply:
This is to assess if it even makes sense to make this change or not